docs/contributing/coding-style.rst

```diff
@@ -1,43 +1,55 @@
 Coding Style
 ============
 
+Saleor uses the `pre-commit <https://pre-commit.com/#install>`_ tool to check and
```

Comment:
When users read this section they'll first read how to use `pre-commit` but we don't say how to install it so I'm worried we'll be getting questions like "pre-commit command doesn't exist, how do I fix it".

At the end of this section, we say:

``python -m pip install -r requirements_dev.txt`` can install these tools. For Pipenv users, they can also be installed by ``pipenv install --dev``.

What about putting it at the beginning like so:

Saleor uses various tools to maintain a common coding style and help with development. To install all the development tools do:

``python -m pip install -r requirements_dev.txt`` 

If you're using `Pipenv`: 

``pipenv install --dev``.

And then we would go with "Saleor uses pre-commit tool to check [...]".